deploy/kubernetes/base/controller.yaml

```diff
@@ -156,6 +156,7 @@ spec:
           imagePullPolicy: IfNotPresent
           args:
             - --csi-address=$(ADDRESS)
+            - --timeout=60s
```

Comment:
Did you measure any other values (i.e. why not 30s?), or is this PR just a quick fix for a common customer pain point.

Reply:
Great question! - measured against 30s and saw a significant improvement compared to the default, but still observed context cancellations for attach and resize operations in several external test runs. For 60s, out of 25 external runs, **0** context cancellations were extracted from the logs.

Not directly relevant to this discussion but for perspective, azuredisk CSI driver sets a timeout value of `1200s` for the external attacher. Having said that, based on that data and given this is not a one way door, I suggest updating the timeout values to 60s as a more sensible default without potentially introducing excessively long waits in cases where an operation is genuinely stalled.

Will post one more comment with a few data points answering outstanding questions before this is merged.